src/Helper/OrganisationApiHelper.php

```diff
@@ -26,167 +30,166 @@ class OrganisationApiHelper {
   /**
    * Constructor.
    */
-  public function __construct(Settings $settings) {
+  public function __construct(Settings $settings, ClientInterface $httpClient) {
     $this->settings = $settings;
-  }
-
-  /**
-   * Get curl client.
-   */
-  private function getCurlClient(): CurlHttpClient {
-    if ($this->client) {
-      return $this->client;
-    }
-    else {
-      $this->client = new CurlHttpClient();
-
-      return $this->client;
-    }
+    $this->httpClient = $httpClient;
   }
 
   /**
    * Get bruger informationer for bruger.
    *
    * @phpstan-return array<string, mixed>
+   *
+   * @throws \Drupal\os2forms_organisation\Exception\ApiException
+   *   API exception.
    */
   public function getBrugerInformationer(string $brugerId): array {
-    $client = $this->getCurlClient();
-
-    $this->settings->getOrganisationApiEndpoint();
+    $response = $this->get('bruger/' . $brugerId);
 
-    $response = $client->request('GET', $this->settings->getOrganisationApiEndpoint() . 'bruger/' . $brugerId);
-
-    if (200 === $response->getStatusCode()) {
-      return $response->toArray();
-    }
-    else {
+    if (Response::HTTP_OK != $response->getStatusCode()) {
```

Comment:
Will this ever happen (without an error being thrown in `get`)?

Reply:
It doesn't appear so. Removed.